.travis.yml

```diff
@@ -4,7 +4,7 @@ cache: pip
 
 python:
   - 3.6
-  - 3.7
+  - 3.7.13 # specify micro version to avoid having EnvCommandError
   - 3.8
   - 3.9
```

Comment:
does travis support 3.10 yet?

Reply:
it supports but I just wanted to focus on ignoring the import error in this PR.